packages/@react-spectrum/label/src/Label.tsx

```diff
@@ -59,9 +65,12 @@ function Label(props: SpectrumLabelProps, ref: DOMRef<HTMLLabelElement>) {
     styleProps.className
   );
 
+  const pressProps = {onClick, onKeyDown, onKeyUp, onMouseDown, onPointerDown, onPointerUp};
+
   return (
     <ElementType
       {...filterDOMProps(otherProps)}
+      {...pressProps}
```

Comment:
instead of pulling out these into a specific pressprops object, lets just pass all the html attributes through to the node
{...otherProps}
this way we don't make assumptions about how usePress works
I think your type changes should already support this.

this is an internal component, so we're not too worried about this deviation from the norm

Reply:
Thats what I was planning on doing, my issue was if I left it on `otherProps` the `filterDOMProps` function removes the handlers, but if I pass `otherProps` without going through `filterDOMProps` then we end up with things like `isDisabled` on the DOM which we don't like.

Reply:
Yep, that is definitely less than ideal. And filterDOMProps no longer lets anything through except id and label props. 
It looks like these are things coming through useProviderProps though, not ones being passed in
Maybe we pull out the props that the Provider adds? or not add them to the props object to begin with? We are using isRequired from it
Also, when disabled, we probably shouldn't try to pass focus from the label to the select. I don't think it'd do anything, but probably still good practice.

@devongovett got a preference?

Reply:
We could probably get away with just a simple `onClick` for this tbh. I don't think we need any of the advanced stuff that usePress offers for this case. Labels aren't keyboard focusable, and we don't need any fancy active state handling. We should test that on touch devices and ensure it works well though.